packages/mc-http-server/bin/start-mc.js

```diff
@@ -21,6 +21,7 @@ if (flags.help) {
   Options:
   --config=<path>           (required) The path to the environment config (defined as a JSON file, e.g. "env.json").
   --csp=<path>              (optional) The path to the custom CSP directives config (defined as a JSON file, e.g. "csp.json").
+  --featurePolicies=<path>  (optional) The path to the feature policy directives config (defined as a JSON file, e.g. "feature-policies.json").
```

Comment:
I'm not sure I like adding more CLI options, especially since this one is very specific.

I'd rather have one general file for specifying custom headers, like `headers.json`. We would then **deprecate** `csp.json` as it will be included in the `headers.json` object as `csp`.

```json
{
  "csp": {},
  "featurePolicies": ""
}
```

I think this would scale better and it would be easier to maintain.

Reply:
Sure. Makes sense to me. Let me give it a shot with a deprecation message.

Reply:
Ok cool! 💯

Reply:
6f576b5 I presume we only use the deprecated file if the path is passed and the headers don't contain CSPs. In other words merging the contents of the deprecated option and the new headers would be odd ot me.